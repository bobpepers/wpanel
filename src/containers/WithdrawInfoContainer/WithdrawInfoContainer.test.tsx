import { shallow } from 'enzyme';
import * as React from 'react';
import { WithdrawInfo } from './';

const defaultProps = {
    match: {
        params: {
            id: '1234',
        },
    },
    withdrawUserHistory: [],
    getWithdrawsInfo: jest.fn(),
    getWithdrawUserHistory: jest.fn(),
};

describe('WithdrawInfo test', () => {
    it('should render', () => {
        const wrapper = shallow(<WithdrawInfo {...defaultProps} />);
        expect(wrapper).toMatchSnapshot();
        expect(wrapper).toBeDefined();
    });
});
