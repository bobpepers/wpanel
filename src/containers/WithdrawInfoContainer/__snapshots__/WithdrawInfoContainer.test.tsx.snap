// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WithdrawInfo test should render 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
